peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -285,7 +297,8 @@ <h1>{{ document.title }}</h1>
                                     <a class="dropdown-item {% if version.pk == document.pk %}active{% endif %}"
                                        rel="alternate"
                                        hreflang="{{ version.language.iso }}"
-                                       href="{{ request.scheme }}://{{ request.get_host }}{{ version.get_absolute_url }}">{% trans version.language.name %} ({{ version.language.iso }})</a>
+                                       href="{{ request.scheme }}://  {{ request.get_host }}{{ version.get_absolute_url }}">{% trans version.language.name %}
```

Comment:
What's going on here? I don't think any of this should be changed?

Reply:
Pycharm linting 🤦🏿‍♂️